### PR TITLE
[Merged by Bors] - feat(CategoryTheory/Galois): prerequisites for essential surjectivity

### PR DESCRIPTION
**Pull request by chrisflav**
This PR collects prerequisites for the essential surjectivity of fiber functors in a Galois category.

---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/RepresentationTheory/Action/Concrete.lean [hunk 1]
```diff
@@ -79,6 +87,74 @@ theorem ofMulAction_apply {G : Type u} {H : FintypeCat.{u}} [Monoid G] [MulActio
...
+      ext (x : G ⧸ N)
+      induction' x using Quotient.inductionOn with x
+      simp
+    map_mul' := fun σ τ ↦ by
+      apply Action.hom_ext
+      ext (x : G ⧸ N)
+      induction' x using Quotient.inductionOn with x
+      show ⟦x * (σ * τ)⁻¹⟧ = ⟦x * τ⁻¹ * σ⁻¹⟧
+      rw [mul_inv_rev, mul_assoc, Subgroup.coe_mul]
+  }
```
Comment (jcommelin) [inline Mathlib/RepresentationTheory/Action/Concrete.lean:127]: I think we should have this map as a standalone definition for the case `G ->* End`. Then you can compose that map with the inclusion `H -> G` inside this definition.

Comment (chrisflav) [reply Mathlib/RepresentationTheory/Action/Concrete.lean]: Done

File: Mathlib/RepresentationTheory/Action/Concrete.lean [hunk 2]
```diff
@@ -79,6 +87,78 @@ theorem ofMulAction_apply {G : Type u} {H : FintypeCat.{u}} [Monoid G] [MulActio
...
+
+/-- Shorthand notation for the quotient of `G` by `H` as a finite `G`-set. -/
+notation:10 G:10 " ⧸ₐ " H:10 => Action.FintypeCat.ofMulAction G (FintypeCat.of <| G ⧸ H)
+
+variable {G : Type*} [Group G] (H N : Subgroup G) [Fintype (G ⧸ N)]
+
+/-- If `N` is a normal subgroup of `G`, then this is the group homomorphism
+sending an element `g` of `G` to the `G`-endomorphism of `G ⧸ₐ N` given by
+multiplication with `g⁻¹` on the right. -/
+def toEndMul [N.Normal] : G →* End (G ⧸ₐ N) where
```
[the remark this answers is not in the document]
Comment (chrisflav) [reply Mathlib/RepresentationTheory/Action/Concrete.lean]: I added this one and one more.

**Review by jcommelin — COMMENTED** [review]
maintainer merge

**Comment by chrisflav** [comment]
Thanks for the reviews!

**Pull request closed without merging**